developer/services/qrexec.md

```diff
@@ -86,25 +86,23 @@ Additionally, disposable VMs are tightly integrated -- RPC to a DisposableVM is
 
 ### Policy files
 
-The dom0 directory `/etc/qubes-rpc/policy/` contains a file for each available RPC action that a VM might call.
+The dom0 directory `/etc/qubes/policy.d/` contains a file for each available RPC action that a VM might call.
```

Comment:
**Nitpicking**: The files in `/etc/qubes/policy.d/` are no longer split by RPC action. Rather they are organised by something more akin to _topics_ and _priority_. 

Since you're updating this sentence it might be a good opportunity to convey that as well? To me, saying "contains a file for each policy" is probably enough, since you're already updating the examples further down.

#### TL;DR:

```suggestion
The dom0 directory `/etc/qubes/policy.d/` contains a file for each available policy.
```

#### Reference docs:

> The biggest difference between the old and new policy formats is that, under the new format, the entire policy is a single entity. It is not divided into separate, per-service fragments. ([source](https://www.qubes-os.org/news/2020/06/22/new-qrexec-policy-system/#policy-files))

#### Example:

(For context, I know you've seen this @skyvine.)

```
# dom0

ls /etc/qubes-rpc/
# admin.backup.Cancel
# admin.backup.Execute
# admin.backup.Info
# admin.deviceclass.List
# ... (long list)
```

```
# dom0

ls /etc/qubes/policy.d

# 30-user.policy
# 35-compat.policy
# 80-whonix.policy
# ... (way shorter list)
```